openhtf/__init__.py

```diff
@@ -45,6 +46,106 @@ class InvalidTestPhaseError(Exception):
   """Raised when an invalid method is decorated."""
 
 
+def Test(*phases, **metadata):
+  return TestInfo(phases, metadata=metadata)
+
+
+class TestInfo(mutablerecords.Record(
```

Comment:
This is not a record; it has an 'Execute' method.  Please use standard Python syntax for class definitions and only use mutablerecord for POD types.